testng-eclipse-plugin/src/main/org/testng/eclipse/ui/TestRunnerViewPart.java

```diff
@@ -229,9 +229,6 @@ public void init(IViewSite site, IMemento memento) throws PartInitException {
     if(progressService != null) {
       progressService.showBusyForFamily(TestRunnerViewPart.FAMILY_RUN);
     }
-
-    // Start the watcher thread, if applicable
-    updateResultThread();

```

Comment:
both `init(IViewSite site, IMemento memento)` and `createPartControl(Composite parent)` are called when create the viewer part, so let's call `updateResultThread()` once and only after the UI control instantiated.